wordle-web/components/settings-dialog.vue

```diff
@@ -1,83 +1,115 @@
 <template>
-  <div class>
-    <v-container @click="toggleDialog">
-      Settings <v-icon>mdi-cog</v-icon>
-    </v-container>
-
-    <v-dialog v-model="dialog" width="450">
-      <v-card>
-        <v-container>
-          <v-card-title>Settings</v-card-title>
-
-          <v-card-text>
-            <v-menu offset-y>
-              <template #activator="{ on, attrs }">
-                <v-btn
-                  color="primary"
-                  class="mb-5"
-                  dark
-                  v-bind="attrs"
-                  v-on="on"
-                >
-                  Pick Theme
-                </v-btn>
-              </template>
-              <v-list>
-                <v-list-item-group>
-                  <v-list-item @click="turnOnTheLights">
-                    <v-list-item-title> Light Mode </v-list-item-title>
-                  </v-list-item>
-                  <v-list-item @click="turnOffTheLights">
-                    <v-list-item-title> Dark Mode </v-list-item-title>
-                  </v-list-item>
-                </v-list-item-group>
-              </v-list>
-            </v-menu>
-
-            <v-menu offset-y>
-              <template #activator="{ on, attrs }">
-                <v-btn color="primary" dark v-bind="attrs" v-on="on">
-                  Change Color Scheme
-                </v-btn>
-              </template>
-              <v-list>
-                <v-list-item-group>
-                  <v-list-item @click="defaultTheme">
-                    <v-list-item-title> Default </v-list-item-title>
-                  </v-list-item>
-                  <v-list-item @click="purpleTheme">
-                    <v-list-item-title> Cotton Candy </v-list-item-title>
-                  </v-list-item>
-                  <v-list-item @click="orangeTheme">
-                    <v-list-item-title> Spooky </v-list-item-title>
-                  </v-list-item>
-                  <v-list-item @click="naturalTheme">
-                    <v-list-item-title> Natural </v-list-item-title>
-                  </v-list-item>
-                </v-list-item-group>
-              </v-list>
-            </v-menu>
-          </v-card-text>
-        </v-container>
-      </v-card>
-    </v-dialog>
+    <div>
+
+      <v-btn
```

Comment:
One big thing to note, is that you guys should only have files that are relevant to the current assignment the PR is for in here. So at some point there should have been a sync with the files in Assignment 3 and your code base to make sure you were up to date with our repo.